src/corespotlight.cs

```diff
@@ -1075,5 +1092,14 @@ interface CSSearchQuery {
 		[Export ("cancel")]
 		void Cancel ();
 	}
+
+	[NoTV, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (NSObject))]
+	interface CSImportExtension : NSExtensionRequestHandling
+	{
+		[Export ("updateAttributes:forFileAtURL:error:")]
+		bool UpdateAttributes (CSSearchableItemAttributeSet attributes, NSUrl contentUrl, [NullAllowed] out NSError error);
```

Comment:
-> `Update`

`attributes` is the first parameter, not the action

In Swift the name is `update`